lib/restaurant.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'menu'
+require 'twilio-ruby'
+#require 'yaml'
+
+class Restaurant
+
+  attr_accessor :order, :menu
+
+  def initialize(menu_class = Menu)
+    @order = []
+    @menu = menu_class.new
+  end
+
+  def display_menu
+  @menu.show_menu.each do |k|
+     "#{k}"
+   end
+  end
+
+  def add_to_order(dish, quantity = 1)
+    dish = dish.capitalize
+    fail "#{dish} isn't available" if dish_check(dish)
```

Comment:
it's not intuitive that `dish_check` should return true if the check fails and false if it succeeds.  This should be:
```ruby
fail "#{dish} isn't available" unless dish_check(dish)
```